task/02-numbers-tasks.js

```diff
@@ -127,7 +134,8 @@ function getAngleBetweenVectors(x1, y1, x2, y2) {
  *     0     => 0
  */
 function getLastDigit(value) {
-    throw new Error('Not implemented');
+	const strValue = value.toFixed();
```

Comment:
try to use remainder operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder